src/fields/vec_seq_double.hh

```diff
@@ -0,0 +1,83 @@
+/*
+ * vec_seq_double.hh
+ *
+ *  Created on: Aug 30, 2012
+ *      Author: jb
+ */
+
+
+#ifndef VECTOR_SEQ_DOUBLE_HH_
+#define VECTOR_SEQ_DOUBLE_HH_
+
+#include <vector>
+#include <memory>
+
+#include <petscvec.h>
+
+#include "fields/field_elementwise.hh"
+
+
+/**
+ * Auxiliary class for output elementwise concentration vectors
+ * in convection transport, sorptions, dual porosity etc.
+ *
+ * Stores data in two formats:
+ *  - shared pointer to std::vector of double
+ *  - pointer to PETSC vector
+ *
+ * Allows the following functionalities:
+ *  - return shared pointer to std::vector of double
+ *  - return pointer to PETSC vector
+ *  - create shared pointer to FieldElementwise object corresponding with std::vector of double
+ */
+class VectorSeqDouble {
+public:
+	typedef typename std::shared_ptr< std::vector<double> > VectorSeq;
+
+	/// Constructor.
+	VectorSeqDouble(unsigned int size)
```

Comment:
Implement no constructor. Turn this constructor into reinit method in order to to avoid unnecessary pointers to VecSeqDouble.